jenkins/job-configs/kubernetes-jenkins-pull/credentials.yaml

```diff
@@ -3,5 +3,5 @@
     wrappers:
         - credentials-binding:
             - file:
-                credential-id: 'kubekins@kubernetes-jenkins.iam.gserviceaccount.com'
+                credential-id: 'pr-kubekins@kubernetes-jenkins-pull.iam.gserviceaccount.com'
```

Comment:
What do you think about changing the id to something like gcp-service-account and making the name of the user transparent to the configs?

Regardless in my attempt to get things working this week I wound up deleting `pr-kubekins@` and replacing the `kubekins@` creds with `pr-kubekins@` ones... So please don't merge this until fixing that.